AMM/contracts/AMM/src/utils.sw

```diff
@@ -0,0 +1,13 @@
+library utils;
+
+use std::{constants::ZERO_B256, contract_id::ContractId, mem::addr_of};
+
+// will remove once this function becomes a part of std-lib
```

Comment:
Could you add a github link to the issue pls

Reply:
I added a link to the merged PR :)